src/components/NewClusterModal.js

```diff
@@ -449,35 +449,35 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
         oldRuntime.masterMachineType !== newRuntime.masterMachineType)
   }
 
-  getImageUrl(clusterDetails) {
-    return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), clusterDetails?.runtimeImages)?.imageUrl
+  getImageUrl(runtimeDetails) {
+    return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), runtimeDetails?.runtimeImages)?.imageUrl
   }
 
   componentDidMount = _.flow(
-    withErrorReporting('Error loading cluster'),
+    withErrorReporting('Error loading environment'),
```

Comment:
Nit: I would maybe say `cloud environment` since that's what we call it to the user.

Reply:
Done.